editor/trackview.cpp

```diff
@@ -96,10 +96,10 @@ void TrackView::updatePalette()
 void TrackView::updateFont(const QFontMetrics &fontMetrics)
 {
 	rowHeight = fontMetrics.lineSpacing();
-	trackWidth = fontMetrics.width('0') * 16;
+	trackWidth = fontMetrics.maxWidth() * 16;
```

Comment:
This isn't *technically* quite the same, but I suspect it doesn't matter in practice, because we're always using monospace fonts here.

But what really troubles me (not a complaint about the change BTW) is that according to the docs, `QFontMetrics::width(QChar ch) const` was already removed in Qt5. But we're building on Qt5 just fine, so what's going on?

Reply:
OK, it seems like this is a bit of a mess... 
The function was made [obsolete](https://doc.qt.io/qt-5/qfontmetrics-obsolete.html#width), but the replacement (`QFontMetrics::whorizontalAdvance(QChar ch) const`) wasn't added until Qt 5.11...

Especially in that light, I think using `maxWidth()` is the right thing to do, because it's been around since at least Qt 4.8, so there's no risk in breaking people's builds. Another option could be `averageCharWidth()`, but I'm not sure that's much of an improvement either.

So yeah, I think this is the right thing to do.